src/app/descriptor-languages/Nextflow.ts

```diff
@@ -7,10 +7,11 @@ export const extendedNFL: ExtendedDescriptorLanguageBean = {
   shortFriendlyName: 'Nextflow',
   friendlyName: 'Nextflow',
   defaultDescriptorPath: '/nextflow.config',
-  descriptorPathPattern: '^^/([^/?:*|<>]+/)*[^/?:*|<>]+.(config)',
+  descriptorPathPattern: '^/([^/?:*|<>]+/)*[^/?:*|<>]+.(config)',
```

Comment:
I don't think the extra hat does anything; but it caused a test to fail when I switched to using the descriptorPathPattern member from Nextflow.ts instead of the nflPath in validationMessages.model.ts